**LabDadosPrev("RPPS") - Laboratório de Análise de Dados Previdenciários**

</br>

# **Municípios com menos de 50 mil habitantes e as transferências voluntárias**
----
"_Sem dados, você é apenas mais uma pessoa com uma opinião_" - **W. E. Deming**

</br>

>[Marcos F Silva](https://audinalytics.netlify.app/)   
**Última Atualização:** 11/07/2021  
**Status:** concluído




## **Introdução**


<!--

As pessoas que militam no segmento de RPPS sabem que quando o Ente está com o Certificado de Regularidade Previdenciária (CRP) vencido fica sujeito a diversas restrições, dentre as quais não ter mais acesso às transferências voluntárias da União. E uma das razões para que um Ente não consiga atualizar o seu CRP é estar inadimplente com as contribuições previdenciárias.

-->

[Notícia recente](https://www.cnm.org.br/index.php/comunicacao/noticias/municipios-de-ate-50-mil-habitantes-nao-dependerao-de-adimplencia-para-receber-transferencia-voluntaria) (01/06/2021) nos informa que municípios com menos de 50 mil habitantes não dependerão de adimplência para receber transferência voluntária. 

A proposta abrange a inadimplência de uma forma bem mais ampla:  *“(...) não dependerão da situação de adimplência do Município de até 50.000 habitantes, identificada em cadastros ou sistemas de informações financeiras, contábeis e fiscais”* 

De fato, a medida alcança todos os municípios com menos de 50 mil habitantes com problemas nos itens do CAUC relativos a `Obrigações de Adimplência Financeira` que abrange os itens 1.1, 1.3, 1.4 e 1.5.

Apresentamos a seguir a descrição da codificação do CAUC:


Código do Item |	Exigência
----------------|-------------------------
**I**	|**Obrigações de Adimplência Financeira** |
1.1	| Regularidade quanto a Tributos, a Contribuições Previdenciárias Federais e à Dívida Ativa da União
1.3	|Regularidade quanto a Contribuições para o FGTS
1.4	|Regularidade em relação à Adimplência Financeira em Empréstimos e Financiamentos concedidos pela União
1.5	|Regularidade perante o Poder Público Federal
**II**|	**Adimplemento na Prestação de Contas de Convênios**|
2.1	|Regularidade quanto à Prestação de Contas de Recursos Federais recebidos anteriormente
2.1.1	|SIAFI/Subsistema Transferências
2.1.2	|Plataforma +Brasil
**III**|	**Obrigações de Transparência**|
3.1	|Relatório de Gestão Fiscal - RGF
3.1.1	|Publicação do Relatório de Gestão Fiscal
3.1.2	|Encaminhamento do Relatório de Gestão Fiscal ao Siconfi
3.2	|Relatório Resumido de Execução Orçamentária - RREO
3.2.1	|Publicação do Relatório Resumido de Execução Orçamentária - RREO
3.2.2	|Encaminhamento do Relatório Resumido de Execução Orçamentária ao Siconfi
3.2.3	|Encaminhamento do Anexo 8 do Relatório Resumido de Execução Orçamentária ao Siope
3.2.4	|Encaminhamento do Anexo 12 do Relatório Resumido de Execução Orçamentária ao Siops
3.3	|Encaminhamento das Contas Anuais
3.4	|Encaminhamento da Matriz de Saldos Contábeis
3.5	|Encaminhamento de Informações para o Cadastro da Dívida Pública - CDP
**IV**|	**Adimplemento de Obrigações Constitucionais ou Legais**|
4.1	|Exercício da Plena Competência Tributária
4.2	|Regularidade Previdenciária
4.3	|Regularidade quanto à Concessão de Incentivos Fiscais
**V**|	**Cumprimento de Limites Constitucionais e Legais**|
5.1	|Aplicação Mínima de recursos em Educação
5.2	|Aplicação Mínima de recursos em Saúde
5.3	|Limite de Despesas com Parcerias Público-Privadas (PPP)
5.4	|Limite de operações de crédito, inclusive por antecipação de receita

A regularidade previdênciária, objeto do item 4.2 e aferida pelo CRP, não parece relacionar-se diretamente com a medida legislativa em questão.

Para mais informações sobre o CAUC, consultar a seguinte [página](https://www.tesourotransparente.gov.br/temas/estados-e-municipios/cauc)

## **Objetivo**

Nosso objetivo inicial é bem simples: identicar quantos Entes com população inferior a 50 mil habitantes estão vinculados ao Regime Próprio de Previdência Social - RPPS e quantos desses pequenos municípios estão regulares no item 4.2 do CAUC que, como pode ser visto na tabela acima, refere-se à regularidade previdenciária.


## **Dados**

Para e execução desta tarefa será necessário obter dois conjuntos de dados:
* Relação dos RPPS brasileiros;    
* População dos municípios brasileiros.

A relação dos RPPS brasileiros consta do site da SPREV. É o arquivo denominado **`Regime Previdenciário Atual dos Entes Federativos`**.

Houve a necessidade de incluir nessa base de dados o **código IBGE** para que fosse possível combinar esta base de dados com outras que possuam esse código como por exemplo o arquivo **`POP2020_20210331.xls`** obtido no site do IBGE contendo a população estimada para os municípios. 

Este arquivo consiste _"(...) de estimativas para 1º de julho de 2020, atualizadas e enviadas ao TCU após a publicação no DOU"_ e pode ser obtido [AQUI](https://www.ibge.gov.br/estatisticas/sociais/populacao/9103-estimativas-de-populacao.html?=&t=resultados).

Um outro conjunto de dados interessante e que possui informações sobre a população dos municípios é o que contém informações sobre o **Sistema de Informações sobre Requisitos Fiscais (CAUC)**.

Estes dados podem ser obtidos no [site do Tesourotransparente](https://www.tesourotransparente.gov.br/ckan/dataset/cauc) e tem como objetivo disponibilizar informações gerenciais do CAUC relativas à situação de cumprimento de requisitos fiscais. 

Outro conjunto de dados que também pode ser interessante trazer para as análises é o **CAPAG dos municípios** também disponível no [site do Tesourotransparente](http://www.tesourotransparente.gov.br/ckan/dataset/capag-municipios).



## **Carregamento dos pacotes necessários**

In [ ]:
#@title
library(readxl)
library(readr)
library(dplyr)
library(stringr)

## **Importação dos dados**

In [ ]:
#@title 
regimeprev <- read_excel("/content/codigo_ibge.xlsx", sheet="crp_final")

In [ ]:
#@title 
pop <- read_excel("/content/POP2020_20210331.xls",
                        sheet="Municípios",
                        range = "A3:E5572",
                        col_names = c("uf", "cod_uf", "cod_munic", "nm_munic", "pop_estimada"))

In [ ]:
#@title 
nomes_colunas <- c("uf", "nm_ente", "cod_ibge", "cod_siafi", "regiao", "populacao", "fonte",
                   "item_1.1", "item_1.3", "item_1.4", "item_1.5", "item_2.1.1", "item_2.1.2",
                   "item_3.1.1", "item_3.1.2", "item_3.2.1", "item_3.2.2", "item_3.2.3", "item_3.2.4",
                   "item_3.3", "item_3.4", "item_3.5", "item_4.1", "item_4.2", "item_5.1", "item_5.2",
                   "item_5.3", "item_5.4")

cauc <- read_csv2("/content/Relatorio-Situacao-de-Varios-Entes---Municipios---UF-Todas---Abrangencia-1.csv",
                  skip=4,
                  na="!",
                  col_names = nomes_colunas,
                  locale=locale(encoding = "latin1"),
                  col_type = cols(),
                  n_max=(5572-4)) 

Feita a importação dos dados, vamos dar uma olhadinha pra ver se tudo correu bem. Se não há nada estranho.

In [ ]:
#@title 
glimpse(regimeprev)

Rows: 5,598
Columns: 8
$ uf             <chr> "AC", "AC", "AC", "AC", "AC", "AC", "AC", "AC", "AC", "…
$ ente           <chr> "Acrelândia", "Assis Brasil", "Brasiléia", "Bujari", "C…
$ cod_uf         <chr> "12", "12", "12", "12", "12", "12", "12", "12", "12", "…
$ nm_estado      <chr> "Acre", "Acre", "Acre", "Acre", "Acre", "Acre", "Acre",…
$ cod_ibge       <chr> "1200013", "1200054", "1200104", "1200138", "1200179", …
$ cnpj           <chr> "84306737000127", "04045993000179", "04508933000145", "…
$ regime         <chr> "RGPS", "RGPS", "RGPS", "RGPS", "RGPS", "RGPS", "RGPS",…
$ dif_ibge_sprev <lgl> NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA,…


In [ ]:
#@title 
glimpse(pop)

Rows: 5,570
Columns: 5
$ uf           <chr> "RO", "RO", "RO", "RO", "RO", "RO", "RO", "RO", "RO", "RO…
$ cod_uf       <dbl> 11, 11, 11, 11, 11, 11, 11, 11, 11, 11, 11, 11, 11, 11, 1…
$ cod_munic    <chr> "00015", "00023", "00031", "00049", "00056", "00064", "00…
$ nm_munic     <chr> "Alta Floresta D'Oeste", "Ariquemes", "Cabixi", "Cacoal",…
$ pop_estimada <chr> "22728", "109523", "5188", "85893", "16204", "15544", "72…


In [ ]:
#@title 
glimpse(cauc)

Rows: 5,568
Columns: 28
$ uf         <chr> "AC", "AC", "AC", "AC", "AC", "AC", "AC", "AC", "AC", "AC",…
$ nm_ente    <chr> "Acrelândia", "Assis Brasil", "Brasiléia", "Bujari", "Capix…
$ cod_ibge   <dbl> 1200013, 1200054, 1200104, 1200138, 1200179, 1200203, 12002…
$ cod_siafi  <chr> "0643", "0157", "0105", "0645", "0647", "0107", "0651", "01…
$ regiao     <chr> "N", "N", "N", "N", "N", "N", "N", "N", "N", "N", "N", "N",…
$ populacao  <dbl> 12538, 6075, 21438, 8474, 8810, 78444, 15126, 32311, 6531, …
$ fonte      <lgl> NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA,…
$ item_1.1   <chr> NA, NA, "15/11/21", NA, NA, "12/06/21", "14/09/21", "20/09/…
$ item_1.3   <chr> NA, NA, "22/08/21", "07/08/21", NA, "22/08/21", "19/08/21",…
$ item_1.4   <chr> "03/06/21", "03/06/21", "03/06/21", "03/06/21", "03/06/21",…
$ item_1.5   <chr> NA, NA, "03/06/21", NA, NA, "03/06/21", "03/06/21", "03/06/…
$ item_2.1.1 <chr> "03/06/21", "03/06/21", "03/06/21", NA, "03/06/21", "03/06/…
$ item_2.1.2 <ch

A variável `cod_ibge` está no formato numérico. Vamos converter para caractere para manter a compatibilidade com o formato dessa variável nas outras bases. 

In [ ]:
#@title 
cauc <- cauc %>% mutate(cod_ibge = as.character(cod_ibge))

Olhando para esses resumos, alguns pontos podem ser observados:
* A quantidade de linhas é diferente nos 3 conjuntos de dados
* A população estimada, no conjunto de dados `pop` foi interpretada como caractere.

Inspecionando a planilha com os dados, verifica-se que em alguns registros, os valores da população estão acompanhados de uma nota. Será necessário fazer a extração das mesmas e converter os dados para o formato numérico.

Mas antes, vamos dar uma checada nesses valores.

In [ ]:
#@title 
pop %>% filter(grepl("\\(.*\\)", pop_estimada))

uf cod_uf cod_munic nm_munic                  pop_estimada
1  RO 11     00205     Porto Velho               539354(1)   
2  AM 13     00607     Benjamin Constant         43935(12)   
3  AM 13     00839     Caapiranga                13283 (13)  
4  AM 13     01654     Guajará                   16937(14)   
5  AM 13     02306     Jutaí                     13886(15)   
6  AM 13     02405     Lábrea                    46882(16)   
7  AM 13     02553     Manaquiri                 33049 (2)   
8  AM 13     03403     Parintins                 115363(17)  
9  AM 13     03601     Santa Isabel do Rio Negro 25865(3)    
10 AM 13     04062     Tabatinga                 67182(18)   
11 AM 13     04302     Urucará                   16130 (4)   
12 AM 13     04401     Urucurituba               23585(19)   
13 PA 15     03754     Jacareacanga              7590 (5)    
14 MA 21     07506     Paço do Lumiar            123747(6)   
15 PE 26     05509     Ferreiros                 12170(7)    
16 BA 29     09208     Coronel João Sá           15717 (8)   
17 BA 29     12004     Ibiassucê                 9031(9)     
18 BA 29     27101     Rodelas                   9442 (10)   
19 BA 29     33208     Vera Cruz                 43716 (11)

Como mencionado, é possível ver as indicações das notas ao lado dos valores da população. 

Vamos remover estas notas e converter os valores para o formato numérico.

In [ ]:
#@title 
pop <- pop %>%
           mutate(pop_estimada = as.integer(str_remove_all(pop_estimada, "\\(.*\\)")))

In [ ]:
#@title 
glimpse(pop)

Rows: 5,570
Columns: 5
$ uf           <chr> "RO", "RO", "RO", "RO", "RO", "RO", "RO", "RO", "RO", "RO…
$ cod_uf       <dbl> 11, 11, 11, 11, 11, 11, 11, 11, 11, 11, 11, 11, 11, 11, 1…
$ cod_munic    <chr> "00015", "00023", "00031", "00049", "00056", "00064", "00…
$ nm_munic     <chr> "Alta Floresta D'Oeste", "Ariquemes", "Cabixi", "Cacoal",…
$ pop_estimada <int> 22728, 109523, 5188, 85893, 16204, 15544, 7220, 18798, 32…


Com as bases de dados importadas, é possível agora combiná-las em uma única base de dados. 

Isso é possível, juntando à base `regimeprev` os dados da população estimada para os municípios constante da base `pop` e também juntar os dados do CAUC que está na base `cauc`.

Nas três bases os municípios são identificados de forma única com o código do IBGE. Na base `pop` precisamos fazer um pequeno  ajuste antes de começar a juntar as bases.

In [ ]:
#@title 
pop <- pop %>% mutate(cod_ibge = paste0(cod_uf, cod_munic))
head(pop, 3)

uf cod_uf cod_munic nm_munic              pop_estimada cod_ibge
1 RO 11     00015     Alta Floresta D'Oeste  22728       1100015 
2 RO 11     00023     Ariquemes             109523       1100023 
3 RO 11     00031     Cabixi                  5188       1100031

In [ ]:
#@title 
dados <- regimeprev %>%
            left_join(select(pop, cod_ibge, pop_estimada), by="cod_ibge") %>%
            left_join(select(cauc, cod_ibge, populacao, regiao, starts_with("item")), by="cod_ibge")

In [ ]:
#@title 
glimpse(dados)

Rows: 5,598
Columns: 32
$ uf             <chr> "AC", "AC", "AC", "AC", "AC", "AC", "AC", "AC", "AC", "…
$ ente           <chr> "Acrelândia", "Assis Brasil", "Brasiléia", "Bujari", "C…
$ cod_uf         <chr> "12", "12", "12", "12", "12", "12", "12", "12", "12", "…
$ nm_estado      <chr> "Acre", "Acre", "Acre", "Acre", "Acre", "Acre", "Acre",…
$ cod_ibge       <chr> "1200013", "1200054", "1200104", "1200138", "1200179", …
$ cnpj           <chr> "84306737000127", "04045993000179", "04508933000145", "…
$ regime         <chr> "RGPS", "RGPS", "RGPS", "RGPS", "RGPS", "RGPS", "RGPS",…
$ dif_ibge_sprev <lgl> NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA,…
$ pop_estimada   <int> 15490, 7534, 26702, 10420, 12008, 89072, 18696, 34884, …
$ populacao      <dbl> 12538, 6075, 21438, 8474, 8810, 78444, 15126, 32311, NA…
$ regiao         <chr> "N", "N", "N", "N", "N", "N", "N", "N", NA, "N", "N", "…
$ item_1.1       <chr> NA, NA, "15/11/21", NA, NA, "12/06/21", "14/09/21", "20…
$ item_1.3      

In [ ]:
#@title 
dados %>%
  filter(is.na(pop_estimada))

uf ente                                     cod_uf nm_estado          
1  AC Governo do Estado do Acre                12     Acre               
2  AL Governo do Estado de Alagoas             27     Alagoas            
3  AM Governo do Estado do Amazonas            13     Amazonas           
4  AP Governo do Estado do Amapá               16     Amapá              
5  BA Governo do Estado da Bahia               29     Bahia              
6  CE Governo do Estado do Ceará               23     Ceará              
7  ES Governo do Estado do Espírito Santo      32     Espírito Santo     
8  GO Governo do Estado de Goiás               52     Goiás              
9  MA Governo do Estado do Maranhão            21     Maranhão           
10 MG Governo do Estado de Minas Gerais        31     Minas Gerais       
11 MS Governo do Estado do Mato Grosso do Sul  50     Mato Grosso do Sul 
12 MT Governo do Estado do Mato Grosso         51     Mato Grosso        
13 PA Governo do Estado do Pará                15     Pará               
14 PB Governo do Estado da Paraíba             25     Paraíba            
15 PE Governo do Estado de Pernambuco          26     Pernambuco         
16 PI Governo do Estado do Piauí               22     Piauí              
17 PR Governo do Estado do Paraná              41     Paraná             
18 RJ Governo do Estado do Rio de Janeiro      33     Rio de Janeiro     
19 RN Governo do Estado do Rio Grande do Norte 24     Rio Grande do Norte
20 RO Governo do Estado de Rondônia            11     Rondônia           
21 RR Governo do Estado de Roraima             14     Roraima            
22 RS Governo do Estado do Rio Grande do Sul   43     Rio Grande do Sul  
23 SC Governo do Estado de Santa Catarina      42     Santa Catarina     
24 SE Governo do Estado de Sergipe             28     Sergipe            
25 SP Governo do Estado de São Paulo           35     São Paulo          
26 TO Governo do Estado do Tocantins           17     Tocantins          
   cod_ibge cnpj           regime dif_ibge_sprev pop_estimada populacao ⋯
1  1200000  63606479000124 RPPS   NA             NA           NA        ⋯
2  2700000  12200176000176 RPPS   NA             NA           NA        ⋯
3  1300000  04312369000190 RPPS   NA             NA           NA        ⋯
4  1600000  00394577000125 RPPS   NA             NA           NA        ⋯
5  2900000  13937032000160 RPPS   NA             NA           NA        ⋯
6  2300000  07954480000179 RPPS   NA             NA           NA        ⋯
7  3200000  27080530000143 RPPS   NA             NA           NA        ⋯
8  5200000  01409580000138 RPPS   NA             NA           NA        ⋯
9  2100000  06354468000160 RPPS   NA             NA           NA        ⋯
10 3100000  18715615000160 RPPS   NA             NA           NA        ⋯
11 5000000  15412257000128 RPPS   NA             NA           NA        ⋯
12 5100000  03507415000144 RPPS   NA             NA           NA        ⋯
13 1500000  05054861000176 RPPS   NA             NA           NA        ⋯
14 2500000  08761124000100 RPPS   NA             NA           NA        ⋯
15 2600000  10571982000125 RPPS   NA             NA           NA        ⋯
16 2200000  06553481000149 RPPS   NA             NA           NA        ⋯
17 4100000  76416940000128 RPPS   NA             NA           NA        ⋯
18 3300000  42498600000171 RPPS   NA             NA           NA        ⋯
19 2400000  08241739000105 RPPS   NA             NA           NA        ⋯
20 1100000  00394585000171 RPPS   NA             NA           NA        ⋯
21 1400000  84012012000126 RPPS   NA             NA           NA        ⋯
22 4300000  87934675000196 RPPS   NA             NA           NA        ⋯
23 4200000  82951229000176 RPPS   NA             NA           NA        ⋯
24 2800000  13128798000101 RPPS   NA             NA           NA        ⋯
25 3500000  46379400000150 RPPS   NA             NA           NA        ⋯
26 1700000  01786029000103 RPPS   NA             NA           NA        ⋯
   item

Agora vamos excluir os Estados e manter apenas os Municípios que possuam RPPS.

In [ ]:
#@title 
dados <- dados %>%
          filter(!is.na(pop_estimada), regime == "RPPS") 

In [ ]:
#@title 
glimpse(dados)

Rows: 2,130
Columns: 32
$ uf             <chr> "AC", "AL", "AL", "AL", "AL", "AL", "AL", "AL", "AL", "…
$ ente           <chr> "Rio Branco", "Arapiraca", "Atalaia", "Barra de Santo A…
$ cod_uf         <chr> "12", "27", "27", "27", "27", "27", "27", "27", "27", "…
$ nm_estado      <chr> "Acre", "Alagoas", "Alagoas", "Alagoas", "Alagoas", "Al…
$ cod_ibge       <chr> "1200401", "2700300", "2700409", "2700508", "2700706", …
$ cnpj           <chr> "04034583000122", "12198693000158", "12200143000126", "…
$ regime         <chr> "RPPS", "RPPS", "RPPS", "RPPS", "RPPS", "RPPS", "RPPS",…
$ dif_ibge_sprev <lgl> NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA,…
$ pop_estimada   <int> 413418, 233047, 47365, 16068, 18338, 4284, 6710, 27356,…
$ populacao      <dbl> 335796, 214067, 44379, 14228, 17076, 4551, 7032, 25780,…
$ regiao         <chr> "N", "NE", "NE", "NE", "NE", "NE", "NE", "NE", "NE", "N…
$ item_1.1       <chr> "24/06/21", "23/06/21", NA, NA, NA, NA, NA, NA, NA, "27…
$ item_1.3      

In [ ]:
#@title 
dados <- dados %>%
          mutate(menos50milhab = ifelse(pop_estimada < 50000, 1, 0))

Quantos municípios com menos de 50 mil hab. possuem RPPS?

In [ ]:
#@title 
dados %>%
  count(menos50milhab) %>%
  mutate(`%` = round( n / sum(n) * 100, 2))

menos50milhab n    %    
1 0              465 21.83
2 1             1665 78.17

78% dos Entes vinculados ao RPPS possuem menos de 50 mil habitantes.

Com os dados do CAUC é possível verificar como está o encaminhamento da Matriz de Saldos Contábeis - MSC pelos Entes que possuem RPPS. Esta obrigação é evidenciada no item 3.4 do CAUC.

In [ ]:
#@title 
# Não comprovaram o encaminhamento da MSC
sum(is.na(dados$item_3.4))
round(sum(is.na(dados$item_3.4)) / nrow(dados) * 100, 2)  

[1] 541

[1] 25.4

25% (541) dos Municípios com RPPS não comprovaram o encaminhamento da MSC. Mas em quais estados estão esses municípios?